kale/utils/initialize_nn.py

```diff
@@ -0,0 +1,29 @@
+# =============================================================================
+# Author: Sina Tabakhi, sina.tabakhi@gmail.com
+# =============================================================================
+
+"""
+Provide methods for initializing neural network parameters (i.e., weights and biases).
```

Comment:
@XianyuanLiu Can you please take a look at this and let me know your opinion? Thanks.

Reply:
Is there a simpler way to avoid having to create this `initialize_nn.py` file? How about directly initializing the parameters in the function [reset_parameters](https://github.com/SinaTabakhi/pykale/blob/mogonet/kale/predict/decode.py#L233), e.g. using 
```python
    def reset_parameters(self) -> None:
        """Initialize the parameters of the model."""
        torch.nn.init.xavier_normal_(self.model.weight)
        torch.nn.init.zeros_(self.model.bias)
```
rather than
```python
    def reset_parameters(self) -> None:
        """Initialize the parameters of the model."""
        self.model.apply(xavier_init)
        self.model.apply(bias_init)
```

Reply:
I agree that it is simpler in that case, but what about when we need to initialize weights and biases in an `nn.Sequential` container for the following code segment?

```python
def __init__(self, num_modalities: int, num_classes: int, hidden_dim: int) -> None:
        super().__init__()

        self.num_modalities = num_modalities
        self.num_classes = num_classes
        self.model = nn.Sequential(
            nn.Linear(pow(self.num_classes, self.num_modalities), hidden_dim),
            nn.LeakyReLU(0.25),
            nn.Linear(hidden_dim, self.num_classes),
        )
        self.reset_parameters()

    def reset_parameters(self) -> None:
        """Initialize the parameters of the model."""
        self.model.apply(xavier_init)
        self.model.apply(bias_init)
```

Here is a PyTorch suggestion for this case:
[Initialising weights and biases in nn.sequential](https://discuss.pytorch.org/t/initialising-weights-in-nn-sequential/76553)

Reply:
Thanks. I didn't notice it was a `nn.Sequential`. How about this one? Use a for loop to find `nn.Linear` in `self.modules()`. Does it work in this case?
```python
    def reset_parameters(self):
        for m in self.modules():
            if isinstance(m, nn.Linear):
                nn.init.xavier_uniform_(m.weight)
                if m.bias is not None:
                    nn.init.zeros_(m.bias)
```